Python/ToonifyImage/toonify.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import shutil # to save image locally
+imagePath=input("Please enter the path of image to toonify : ")
+r = requests.post(
+    "https://api.deepai.org/api/toonify",
+    files={
+        'image': open(imagePath, 'rb'),
+    },
+    headers={'api-key': 'e6c65153-10a5-45f8-a24e-9d8d7ef4f76f'}

```

Comment:
Never share your actual API keys to anyone or upload it to Github. it can be misused. please replace it with something like `"*********'` or `'Your_API_KEY'`. please take a note of it while dealing with API keys. @Tejas1510